### PR TITLE
Brace expansion

### PR DESCRIPTION
**Pull request by paxperscientiam**
Hey @dylanaraps 

Is this closer to a proper contribution?

### CODE REVIEW

**Comment by dylanaraps** [comment]
You've done it the wrong way around! Pop those in the README.md under a new section. Tests aren't needed as they're just syntax usage. :)

**Comment by paxperscientiam** [comment]
@dylanaraps now? :)

File: README.md [hunk 1]
```diff
@@ -674,6 +674,11 @@ for dir in ~/Downloads/*/; do
     printf '%s\n' "$dir"
 done
 
+# Leverage brace expansion to loop, explicit reference
+for file in /path/to/parentdir/{fil1,file2,subdir/file3}; do
```
Comment (dylanaraps) [inline README.md]: Fix `fil1` --> `file1`

**Comment by dylanaraps** [comment]
Cheers :+1:

**Pull request merged**